tensorflow/contrib/rnn/BUILD

```diff
@@ -27,7 +27,7 @@ load(
 tf_custom_op_py_library(
     name = "rnn_py",
     srcs = ["__init__.py"] + glob(["python/ops/*.py"]) 
-                           + glob(["python/tools/*.py"]),
+                           + glob(["python/tools/.py"]),
```

Comment:
Well, that was embarrassing. Changed it to the right filename and verified that the script actually gets included in the pip package this time around.